scss/components/_tiles.scss

```diff
@@ -0,0 +1,23 @@
+// Grid items that are also links
+
+.tile {
+  background-color: $gray-lightest;
+  border-top: 1rem solid $primary;
+  border-bottom: none;
+  display: block;
+  padding: 2rem 1rem;
```

Comment:
Can we increase the left/right padding to 1.5rem please? Feels a little tight on the sides.